Modules/rat_rescue.py

```diff
@@ -745,7 +745,8 @@ def rats(self, value):
         else:
             raise TypeError(f"expected type list got {type(value)}")
 
-    async def add_rat(self, name: str = None, guid: UUID or str = None, rat: Rat = None) -> None:
+    async def add_rat(self, name: str = None, guid: UUID or str = None, rat: Rat = None) \
+            -> Optional[Rat]:
```

Comment:
i assume this is on a newline due to pep8? i would rather you place the newline after a paramater than during the return type hint.